### DISCUSSION TITLE
SH1106_NONAME_2ND_HW_I2C Teensy 4.0

### DISCUSSION THREAD

**Discussion by GeorgBeckmann** [olikraus/u8g2#1468] [Q&A]
Hi,
I tried to use the 2nd hw i2c for teensy.
There was no output on pin 18-data 17 clk.

When I use a Arduino Due, the code works .
Any suggestions ?
[HelloWorld_Teensy_Be.zip](https://github.com/olikraus/u8g2/files/6332015/HelloWorld_Teensy_Be.zip)

**Comment by olikraus** [comment]
2nd hw i2c is probably not supported :-(

**Reply by GeorgBeckmann** [reply]
Thanks for fast answer. So I didn't make something wrong. 
But I wonder, because the Wire lib with Wire1 works with teensy and
as far as I undersood, u8g2 uses the Wire lib.
